packages/webapp/public/locales/en/expense.json

```diff
@@ -36,8 +36,8 @@
     "CUSTOM_DESCRIPTION": "Recurring expenses related to electricity, gas, water (including irrigation), garbage collection, and other periodic services."
   },
   "LABOUR": {
-    "EXPENSE_NAME": "Labour",
-    "CUSTOM_DESCRIPTION": "Expenses related to employed and contracted individuals."
+    "EXPENSE_NAME": "Contract labour",
```

Comment:
I talked to Loïc and changed this to Contract labour so that we can have two separate filters, one for the regular Labour expenses derived from tasks and one for the one-offs. We were initially thinking to combine them together in the same filter, but since we're not summing them together in the total in the KPI card that seemed more confusing